src/main/java/edu/cornell/kfs/module/purap/CUPurapConstants.java

```diff
@@ -21,6 +21,8 @@ public class CUPurapConstants {
     public static final String PAYMENT_METHODL_REVIEW = "PTMA";
     public static final String B2B_HIGHER_LIMIT_PERMISSION = "B2B Higher APO Limit";
 
+    public static final int REQUISITION_ITEM_DESCRIPTION_MAX_LENGTH = 254;

```

Comment:
If this constant is only being used by unit tests, you may want to move it to the CuPurapTestConstants class to avoid confusion.